Disable the check that dbus is running, since MacPorts dbus is
autolaunched by launchd.
--- src/dbusbind.c.orig	2022-03-11 01:04:22.000000000 -0600
+++ src/dbusbind.c	2022-04-08 18:34:15.000000000 -0500
@@ -303,9 +303,6 @@
 	      || EQ (bus, QCsession_private)))				\
 	  XD_SIGNAL2 (build_string ("Wrong bus name"), bus);		\
 	/* We do not want to have an autolaunch for the session bus.  */ \
-	if ((EQ (bus, QCsession) || EQ (bus, QCsession_private))	\
-	    && session_bus_address == NULL)				\
-	  XD_SIGNAL2 (build_string ("No connection to bus"), bus);	\
       }									\
   } while (0)
 
